src/log/console_appender.cpp

```diff
@@ -20,7 +20,7 @@ namespace fc {
    class console_appender::impl {
    public:
      config                      cfg;
-     boost::mutex                log_mutex;
+     boost::mutex                log_mutex;

```

Comment:
Remove extraneous line of code.

Reply:
Thanks for spotting this. I am not sure what happened here. I will take a look.

Reply:
It seems it's adding carriage return. I am using mac and this never happened to me before. I didn't even touch this line TBH, I just changed the other line. Any ideas?

Reply:
I can't find a way to fix this. It's really just adding a carriage return line at the end of it, but I can't really remove this because my editor didn't add it so I can't remove it. I am not sure how this ended up there. 

Also, I didn't edit this line. So weird.